### PR TITLE
Update emit.json to include more common doodads

### PR DESCRIPTION
**Pull request by pisskop**
Adds in sludge effects, acid dribble effect, fire trail effect (unused, an example), fabric-reality rip, bloated zombie gas puff, spider web-poo, and adds them to critters.

We could replace all field effects with this.

### CODE REVIEW

File: data/json/emit.json [hunk 1]
```diff
@@ -19,5 +19,49 @@
     "//": "Steady stream of tear gas (example: tear gas canister)",
     "field": "fd_tear_gas",
     "qty": 18
+  },{
```
Comment (mugling) [inline data/json/emit.json]: JSON style

File: data/json/emit.json [hunk 2]
```diff
@@ -19,5 +19,49 @@
...
+    "id": "emit_gas_plume",
+    "type": "emit",
+    "//": "occassional wafts of toxic gas (example: gasbag)",
+    "field": "fd_toxic_gas",
+    "qty": 3,
+    "chance": 20
+  },{
+    "id": "emit_fire_small",
+    "type": "emit",
+    "//": "starts fires in its path and/or trail (example: creature made of fire)",
```
Comment (mugling) [inline data/json/emit.json]: Needs better example

**Comment by mugling** [comment]
Although some of those ideas are reasonable I would strongly prefer to migrate existing field sources to `emit` before adding new ones. If you want to float some of them as ideas for later more general inclusion your mod could be an ideal showcase for this.

**Comment by pisskop** [comment]
Fair enough.

Nothing here is out of place in the mod, and more testing will let me smooth out the frequences of the emissions.

**Comment by pisskop** [comment]
Although you should be aware that adding duplicate ids simply overwrites the previous ones.

**Pull request closed without merging**